src/Sparkfabrik/Command/Gitlab/GitlabSearchMergeRequestCommand.php

```diff
@@ -204,11 +204,19 @@ private function handleAgumentStory($story = null)
      *
      * @param string|integer $project_id
      *
-     * @return integer|boolean
+     * @return integer| ConsoleOutput output.
      */
     private function handleAgumentProjectId()
     {
-        return $this->getService()->getConfig()['project_id'];
+        $conf_project_id = $this->getService()->getConfig()['project_id'];
+        if (isset($project_id) && !is_integer($project_id)) {
+            $project_id = $this->findProjectId($project_id);
+        } else if (isset($conf_project_id) &&
+        !is_integer($conf_project_id)) {
+            $project_id = $this->findProjectId($conf_project_id);
+        } else {
+            return ($project_id ? $project_id : $conf_project_id);
+        }
```

Comment:
`handleAgumentProjectId` seems to be shared with `GitlabMergeRequestCommand.php` maybe they can be moved in a parent class ?